packages/neon-js/src/experimental/helpers.ts

```diff
@@ -1,6 +1,8 @@
 import { CONST, rpc, sc, tx, u, wallet } from "@cityofzion/neon-core";
 import { CommonConfig } from "./types";
 import { GASContract } from "./nep17";
+import { OpCode } from "@cityofzion/neon-core/lib/sc";
```

Comment:
Please use proper imports. This is a common mistake within monorepo setups. You should be referencing sc.OpCode instead.

Reply:
Yeah sorry, this is my IDE trying to be smart. I don't even notice that I couldn't call it directly, my IDE automatically adds the imports.